cinema/models.py

```diff
@@ -60,6 +60,10 @@ class Meta:
     def __str__(self):
         return self.movie.title + " " + str(self.show_time)
 
+    @property
+    def tickets_available(self):
+        return self.cinema_hall.capacity - self.tickets.count()
```

Comment:
The `tickets_available` property assumes that the `tickets` related name is correctly set in the `Ticket` model's `movie_session` ForeignKey. Ensure that the `related_name` is correctly defined as `tickets` in the `Ticket` model to avoid any attribute errors.